src/components/ToggleSwitch.js

```diff
@@ -1,89 +1,59 @@
 const React = require('react');
 const Radium = require('radium');
 const StyleConstants = require('../constants/Style');
+const Icon = require('./Icon');
 
 const ToggleSwitch = React.createClass({
   propTypes: {
-    activeColor: React.PropTypes.string,
-    children: React.PropTypes.node,
-    componentStyle: React.PropTypes.oneOfType([
-      React.PropTypes.array,
-      React.PropTypes.object
-    ]),
-    defaultPosition: React.PropTypes.oneOf(['left', 'right']),
-    inactiveColor: React.PropTypes.string,
-    labelStyle: React.PropTypes.oneOfType([
-      React.PropTypes.array,
-      React.PropTypes.object
-    ]),
+    checked: React.PropTypes.bool,
+    checkedByDefault: React.PropTypes.bool,
+    falseIcon: React.PropTypes.element,
```

Comment:
I meant the name of the prop. Instead of naming it `falseIcon`, we could name it `leftIcon` to make it more consistent with the `leftLabel` and `rightLabel` props.